.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
```

Comment:
no need for this

Reply:
But we will need it for integration tests (as it will interact with NM). Do you suggest changing it only when we will need it?

Reply:
Let's add it once we fail with forbidden.